dev-docs/publisher-api-reference.md

```diff
@@ -582,6 +583,23 @@ Use this method to get all of the bid requests that resulted in a NO_BID.  These
 
 <hr class="full-rule">
 
+<a name="module_pbjs.getNoBidsForAdUnitCode"></a>
+
+### pbjs.getNoBidsForAdUnitCode(adUnitCode) ⇒ `Object`
+
+Returns bid requests that resulted in a NO_BID for the specified adUnitCode.  See full documentation at [pbjs.getNoBids()](#module_pbjs.getNoBids).
+
+**Kind**: static method of [pbjs](#module_pbjs)
+
+**Returns**: `Object` - NO_BID bidResponse object
```

Comment:
`getNoBids()` returns an array of objects. I would assume this one could too? Also - the table below should be labeled "parameters".

Reply:
I set this function up to be similar to getBidResponsesForAdunit

Reply:
If they're set up the same, shouldn't the description be the same? They're not. getNoBids says it's an array. This says an object.